error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/SpecifyLocale.java

```diff
@@ -0,0 +1,63 @@
+package tech.picnic.errorprone.bugpatterns;
+
+import static com.google.errorprone.BugPattern.LinkType.CUSTOM;
+import static com.google.errorprone.BugPattern.SeverityLevel.WARNING;
+import static com.google.errorprone.BugPattern.StandardTags.LIKELY_ERROR;
+import static com.google.errorprone.matchers.method.MethodMatchers.instanceMethod;
+import static tech.picnic.errorprone.bugpatterns.util.Documentation.BUG_PATTERNS_BASE_URL;
+
+import com.google.auto.service.AutoService;
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.bugpatterns.BugChecker;
+import com.google.errorprone.bugpatterns.BugChecker.MethodInvocationTreeMatcher;
+import com.google.errorprone.fixes.Fix;
+import com.google.errorprone.fixes.SuggestedFix;
+import com.google.errorprone.matchers.Description;
+import com.google.errorprone.matchers.Matcher;
+import com.sun.source.tree.ExpressionTree;
+import com.sun.source.tree.MethodInvocationTree;
+import tech.picnic.errorprone.bugpatterns.util.SourceCode;
+
+/**
+ * A {@link BugChecker} that flags {@link String#toLowerCase()} or {@link String#toUpperCase()}
+ * which do not specify a Locale.
+ */
+@AutoService(BugChecker.class)
+@BugPattern(
+    summary =
+        "Specify `Locale.ROOT` or `Locale.getDefault()` when calling `String#to{Lower,Upper}Case` without a specific Locale",
+    link = BUG_PATTERNS_BASE_URL + "SpecifyLocale",
+    linkType = CUSTOM,
+    severity = WARNING,
+    tags = LIKELY_ERROR)
```

Comment:
Not sure if `LIKELY_ERROR` or `FRAGILE_CODE` is more appropriate here

Reply:
Would go for `WARNING` and `FRAGILE_CODE`, since the default often _usually_ does what one wants.